webapp/channels/src/actions/invite_actions.ts

```diff
@@ -68,7 +68,7 @@ export function sendMembersInvites(teamId: string, users: UserProfile[], emails:
                     data: emails.map((email) => ({
                         email,
                         error: {error: localizeMessage('invite.members.unable-to-add-the-user-to-the-team', 'Unable to add the user to the team.')},
-                    })) as any, // HARRISONTODO These error handling cases return slightly different types
+                    })) as unknown as TeamInviteWithError[],
```

Comment:
Is there any better way to do this? `as unknown as X` is something that can be done, but it is one of those things of "forget any typing, just do what I want you to do" that we should avoid whenever possible.

Reply:
There would be by actually changing the code logic to make a more accurate object for `error`, but I spent a few minutes trying to figure out how to do that, and I couldn't make it work, so I gave up on it. The code itself behaves as it always has, but now has a scary looking type to make that more obvious instead of using a simple `any`